classes/press/PressDAO.inc.php

```diff
@@ -112,55 +81,26 @@ function deleteAllPubIds($pressId, $pubIdType) {
 	 */
 	function anyPubIdExists($pressId, $pubIdType, $pubId,
 			$assocType = ASSOC_TYPE_ANY, $assocId = 0, $forSameType = false) {
-				$pubObjectDaos = array(
-						ASSOC_TYPE_SUBMISSION => Application::getSubmissionDAO(),
-						ASSOC_TYPE_CHAPTER => DAORegistry::getDAO('ChapterDAO'),
-						ASSOC_TYPE_REPRESENTATION => Application::getRepresentationDAO(),
-						ASSOC_TYPE_SUBMISSION_FILE => DAORegistry::getDAO('SubmissionFileDAO')
-				);
-				if ($forSameType) {
-					$dao = $pubObjectDaos[$assocType];
-					$excludedId = $assocId;
-					if ($dao->pubIdExists($pubIdType, $pubId, $excludedId, $pressId)) return true;
-					return false;
-				}
-				foreach($pubObjectDaos as $daoAssocType => $dao) {
-					if ($assocType == $daoAssocType) {
-						$excludedId = $assocId;
-					} else {
-						$excludedId = 0;
-					}
-					if ($dao->pubIdExists($pubIdType, $pubId, $excludedId, $pressId)) return true;
-				}
-				return false;
-	}
-
-	//
-	// Private functions
-	//
-	/**
-	 * Get the table name for this context.
-	 * @return string
-	 */
-	protected function _getTableName() {
-		return 'presses';
-	}
-
-	/**
-	 * Get the table name for this context's settings table.
-	 * @return string
-	 */
-	protected function _getSettingsTableName() {
-		return 'press_settings';
-	}
-
-	/**
-	 * Get the name of the primary key column for this context.
-	 * @return string
-	 */
-	protected function _getPrimaryKeyColumn() {
-		return 'press_id';
+		$pubObjectDaos = array(
+				ASSOC_TYPE_SUBMISSION => Application::getSubmissionDAO(),
```

Comment:
(Our indentation conventions in PHP are to use one indent. I think this is an inconsistency from already-existing code, though, and this is strictly a pedantic comment.)